README.md

```diff
@@ -199,6 +199,13 @@ in the *AWS Command Line Interface User Guide*.
 The credentials must have a policy applied that
 [allows access to Amazon ECR](http://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr_managed_policies.html).
 
+### Amazon ECR Docker Credential Helper 
+
+| Environment Variable  | Default Value | Description                                                       |
+| --------------------- | ------------- | ----------------------------------------------------------------- |
+| AWS_ECR_DISABLE_CACHE | 1             | Disables the local file auth cache if set to a zero length string |
+| AWS_ECR_CACHE_DIR     | ~/.ecr        | Customises the local file auth cache directory location           |
```

Comment:
nit: We're using American English spelling in this repository, but we can avoid the differences by picking a different word.

```suggestion
| AWS_ECR_CACHE_DIR     | ~/.ecr        | Specifies the local file auth cache directory location           |
```